index.bs

```diff
@@ -638,3 +646,134 @@ registerAnimator('spring', class {
     }
 });
 </pre>
+
+
+Example 2: Twitter header. {#example-2}
+--------------------------
+
+
+<pre class='lang-markup'>
+&lt;style&gt;
+#scroller {
+  --animator-root: twitter-header;
+}
+.avatar {
+  --animator: twitter-header avatar;
+}
+.bar {
+  --animator: twitter-header bar;
+}
+&lt;/style&gt;
+
+&lt;div id="scroller"&gt;
+  &lt;section class="bar"&gt;&lt;/section&gt;
+  &lt;header&gt;
+    &lt;picture class="avatar"&gt;
+      &lt;img src="avatar.jpg"&gt;
+    &lt;/picture&gt;
+    &lt;section&gt;
+      &lt;button&gt;Friends&lt;/button&gt;
+      &lt;button&gt;Edit Profile&lt;/button&gt;
+    &lt;/section&gt;
+  &lt;/header&gt;
+&lt;/div&gt;
+
+&lt;script&gt;
+document.animationWorklet.import('tweeter-header.js');
+&lt;/script&gt;
+
+</pre>
+
+<pre class='lang-javascript'>
+// Inside AnimationWorkletGlobalScope.
+registerAnimator('twitter-header', class {
+
+  static elements =  [
+    {name: 'avatar', inputProperties: [], outputProperties: ['transform']},
+    {name: 'bar', inputProperties: [], outputProperties: ['opacity']}
+  ];
+
+  static timelines = [ 'scroll' ];
+
+  static INIT_HEIGHT = 140;
+
+  animate(elementMap, timelines) {
+    var scrollPos = timelines[0].currentTime;
```

Comment:
Fixed. Note that in the other example I choose to use a constant scroll size. 

I think computing the scroll size and passing it as an input is a nice trick but I prefer to keep the example to the point and avoid unnecessary complexity to make it more realistic. As an aside, perhaps ScrollTimeline should expose its original domain size to help convert from time<->scroll position. Any positioning effect that uses transform can take advantage of that.